.github/styles/Vocab/Products/accept.txt

```diff
@@ -98,31 +198,81 @@ relayer
 Relayer
 relayers
 Relayers
+remix's
+renounceOwnership
```

Comment:
Is this in code?

Reply:
Yes. It's a variable name which is called out in a markdown file.

Reply:
Resolved